src/main/java/com/minecolonies/entity/pathfinding/AbstractPathJob.java

```diff
@@ -895,7 +913,8 @@ protected boolean isWalkableSurface(Block block, BlockPos pos)
         return block.getMaterial().isSolid() &&
                 !(block instanceof BlockFence) &&
                 !(block instanceof BlockFenceGate) &&
-                !(block instanceof BlockWall);
+                !(block instanceof BlockWall) &&
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) '&&' should be on a new line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AOperator_Wrap)